widget/embedded/src/components/Layout/Layout.styles.ts

```diff
@@ -1,19 +1,44 @@
 import { styled } from '@rango-dev/ui';
 
 export const Container = styled('div', {
-  height: '100%',
-  borderRadius: '$xl',
+  position: 'relative',
+  display: 'flex',
+  flexDirection: 'column',
+  //TODO: add radii theme token for 20px
```

Comment:
Yes, other components will utilize this border-radius size; hence, we should modify other tokens for border-radius and give it a meaningful name. this can lead to numerous changes in other components.